packages/vite-plugin-cloudflare/src/utils.ts

```diff
@@ -1,5 +1,27 @@
+import * as path from 'node:path';
 import { invariant } from './shared';
-import type { WorkerOptions } from 'miniflare';
+import type { SharedOptions, WorkerOptions } from 'miniflare';
+
+const DEFAULT_PERSIST_PATH = '.wrangler/state/v3';
```

Comment:
Nit: I thought your preference here was to always rely on `path.join` instead of manually creating paths with `/`?

Reply:
I think it's fine if the paths are static. I just think if you're joining paths then you might as well use `path.join` rather than template strings. I can use `path.join` if you prefer though.

Reply:
I see... I don't really have a strong preference, I just tend to prefer consistency across a codebase (even if it can make some lines a bit more verbose than necessary? 🤔)